i18n/ui.json

```diff
@@ -1,6 +1,9 @@
 {
   "homepage": {
-    "title": "Create accessible React apps<1> with speed</1>",
+    "title": {
+      "main": "Create accessible React apps",
+      "highlighted": "with speed"
+    },
```

Comment:
We should keep the templated version (`<1> with speed</1>`) just in case. We're not using i18n yet, but we'll need it in this format later on. I'll push the updates now, just letting you know.